src/main/java/org/apache/maven/plugins/dependency/fromDependencies/BuildClasspathMojo.java

```diff
@@ -366,7 +372,10 @@ protected String readClasspathFile()
             return null;
         }
         StringBuilder sb = new StringBuilder();
-        try ( BufferedReader r = new BufferedReader( new FileReader( outputFile ) ) )
+        String encoding = outputEncoding == null ? "UTF-8" : outputEncoding;
```

Comment:
I'd copy the same method from MSITE to here.

Reply:
Maybe just

```
        String encoding = Objects.toString( outputEncoding, "UTF-8" );
```